pyteal/ast/int.py

```diff
@@ -64,5 +64,8 @@ def __str__(self):
     def type_of(self):
         return TealType.uint64
 
+    def __hash__(self) -> int:
+        return hash(str(self))
```

Comment:
Why is this needed?

Reply:
it *was* needed, but the implementation changed, and we should remove it